diff --git a/node_modules/@shopify/app-bridge-react/dist/index.js b/node_modules/@shopify/app-bridge-react/dist/index.js
index 915a9ff..ca5294e 100644
--- a/node_modules/@shopify/app-bridge-react/dist/index.js
+++ b/node_modules/@shopify/app-bridge-react/dist/index.js
@@ -15,7 +15,7 @@ function dr() {
   if (Oe)
     return L;
   Oe = 1;
-  var R = Se, v = Symbol.for("react.element"), y = Symbol.for("react.fragment"), c = Object.prototype.hasOwnProperty, C = R.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactCurrentOwner, x = { key: !0, ref: !0, __self: !0, __source: !0 };
+  var R = Se, v = Symbol.for("react.element"), y = Symbol.for("react.fragment"), c = Object.prototype.hasOwnProperty, C = R.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE.ReactCurrentOwner, x = { key: !0, ref: !0, __self: !0, __source: !0 };
   function h(a, d, b) {
     var m, T = {}, w = null, E = null;
     b !== void 0 && (w = "" + b), d.key !== void 0 && (w = "" + d.key), d.ref !== void 0 && (E = d.ref);
@@ -48,7 +48,7 @@ function vr() {
       var r = k && e[k] || e[W];
       return typeof r == "function" ? r : null;
     }
-    var O = R.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
+    var O = R.__CLIENT_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE;
     function p(e) {
       {
         for (var r = arguments.length, t = new Array(r > 1 ? r - 1 : 0), n = 1; n < r; n++)
